eb_hooks.py

```diff
@@ -59,9 +59,16 @@ def parse_hook(ec, *args, **kwargs):
     # determine path to Prefix installation in compat layer via $EPREFIX
     eprefix = get_eessi_envvar('EPREFIX')
 
+#Disable Qt5/5.15.2 qtwebengine in Sanity-check
+    Qt5_check_qtwebengine_disable(ec, eprefix)
+
     if ec.name in PARSE_HOOKS:
         PARSE_HOOKS[ec.name](ec, eprefix)
 
+def Qt5_check_qtwebengine_disable(ec, eprefix):
```

Comment:
@TopRichard Let's rename this to `parse_hook_Qt5_check_qtwebengine_disable`, and register it in `PARSE_HOOKS`, so we keep things consistent.

Done in https://github.com/TopRichard/bot-software-layer1/pull/13